MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,451 @@
+---
+layout: post
+title: Commit in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the commit support in Syncfusion .NET MAUI DataForm (SfDataForm) and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Editing in .NET MAUI DataForm (SfDataForm)
+
+The data form supports several built-in editors.
+
+## Supported editors and associated DataFormItem
+
+<table>
+<tr>
+<th>Editor name</th>
+<th>Editor class</th>
+<th>Data Type/Attribute</th>
+<th>Input control loaded</th>
+</tr>
+<tr>
+<td>
+Text
+</td>
+<td>
+{{'[DataFormTextEditor]'| markdownify }}
+</td>
+<td>
+The String type property and any other type apart from the following specified cases.
+</td>
+<td>
+{{'[Entry]'| markdownify }}
```

Comment:
Include links for all controls used for editors.
like
Entry, Editor, SfComboBox